src/main/java/ladder/domain/RandomLineGenerator.java

```diff
@@ -17,13 +17,13 @@ public Line create(int lineLength) {
     }
 
     private Line createAfterSecondPoints(Line line, int pointCount) {
-        Line addedLine = line;
+        Line accumulatedLine = line;
         while (pointCount > 0) {
             pointCount--;
-            boolean nextPoint = addedLine.isLastPointTrue() ? false : getRandomBoolean();
-            addedLine = addedLine.addPoint(nextPoint);
+            boolean nextPoint = accumulatedLine.isLastTrue() ? false : getRandomBoolean();
```

Comment:
3항 연산자는 안쓰는게 좋습니다.
아울러 
```suggestion
            boolean nextPoint = !accumulatedLine.isLastTrue() && getRandomBoolean();
```
이런식으로도 작성할수 있을듯 해요~

Reply:
추천해주신것과 같이 intellij에서 변경하는것을 추천했었지만 저는 3항연산자가 조금더 일반적으로 다른사람이 이해하기 쉽지않을까 생각했습니다.
현업이나 이런곳에서는 추천해주신 방법이 조금더 많이쓰이면서 잘 읽히는 코드일까요??

Reply:
저는 BooleanExpression이 훨씬 잘 읽히는 코드라고 생각합니다.
명확하게 옆에서 옆으로 읽으면 어떤 느낌인지 알수 있는거 같아요.
사실 이런 코드를 많이봐서 익숙한거 같기도 하고요!